### PR TITLE
update packages --force upgrade

### PR DESCRIPTION
**Pull request by jonahwilliams**
## Description

As it says

### CODE REVIEW

**Review by jacob314 — APPROVED** [review]
lgtm

**Review by grouma — APPROVED** [review]

File: packages/flutter_tools/pubspec.yaml [hunk 1]
```diff
@@ -51,11 +51,13 @@ dependencies:
...
   code_builder: 3.2.0
   build: 1.1.6
-  build_modules: 2.4.3
+  build_modules: 2.5.0
   build_daemon: 2.0.0
-  build_web_compilers: 2.2.3
+  build_web_compilers: 2.3.0
 
   analyzer: 0.36.4 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
+  ansi_up: 0.0.1+4.0.4 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
```
Comment (zanderso) [inline packages/flutter_tools/pubspec.yaml]: Do you need to update the Fuchsia BUILD.gn file for the tool?

**Comment by zanderso** [comment]
FYI @cbracken in case the Fuchsia roll is affected.

**Comment by jonahwilliams** [comment]
Ahh, I didn't see the new dep. Will update

**Comment by jonahwilliams** [comment]
Actually I think since this is transitive we'll be okay, we don't include all of the transitive deps in build.gn

**Pull request merged**